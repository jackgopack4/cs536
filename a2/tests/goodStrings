"this is a good string"
 "this is a good\nstring too"
  "this is also good\"\""
   "I think this should be good too \n\t\'\"\nhihohum"
    ""
     "&!88"
      "use \n to denote a newline character"
       "include a quote like this \" and a backslash like this \\"
