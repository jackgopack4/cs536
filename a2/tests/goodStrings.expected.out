Token: STRINGLITERAL
LineNum: 1
CharNum: 1
Value: "this is a good string"

Token: STRINGLITERAL
LineNum: 2
CharNum: 2
Value: "this is a good\nstring too"

Token: STRINGLITERAL
LineNum: 3
CharNum: 3
Value: "this is also good\"\""

Token: STRINGLITERAL
LineNum: 4
CharNum: 4
Value: "I think this should be good too \n\t\'\"\nhihohum"

Token: STRINGLITERAL
LineNum: 5
CharNum: 5
Value: ""

Token: STRINGLITERAL
LineNum: 6
CharNum: 6
Value: "&!88"

Token: STRINGLITERAL
LineNum: 7
CharNum: 7
Value: "use \n to denote a newline character"

Token: STRINGLITERAL
LineNum: 8
CharNum: 8
Value: "include a quote like this \" and a backslash like this \\"

